API/Entities/EAuth/AuthController.cs

```diff
@@ -5,18 +5,28 @@
 
 namespace AlpimiAPI.Entities.EAuth
 {
+    ///
     [Route("api/[controller]")]
     [ApiController]
+    [Consumes("application/json")]
+    [Produces("application/json")]
     public class AuthController : ControllerBase
     {
         private readonly IMediator _mediator;
 
+        ///
         public AuthController(IMediator mediator) => _mediator = mediator;
 
+        /// <summary>
+        /// Returns JWT token after entering correct Login and Password
```

Comment:
Żeby konwencja zgadzała się z pozostałymi opisami
```suggestion
        /// Returns a JWT token
```